### PR TITLE
Address a performance issue and adjust some rubocop names

### PR DESCRIPTION
**Pull request by chills42**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -12,9 +12,6 @@ RSpec/MultipleExpectations:
 RSpec/NestedGroups:
   Max: 5
 
-RSpec/NotToNot:
```
Comment (pointlessone) [inline .rubocop.yml]: Why was this removed?

File: lib/prawn/soft_mask.rb [hunk 2]
```diff
@@ -75,7 +75,7 @@ def soft_mask(&block)
         renderer.add_content "/#{soft_mask_registry[registry_key]} gs"
       else
         masks = page.resources[:ExtGState] ||= {}
-        id = masks.empty? ? 'GS1' : masks.keys.sort.last.succ
+        id = masks.empty? ? 'GS1' : masks.keys.max.succ
```
Comment (pointlessone) [inline lib/prawn/soft_mask.rb]: This should be in a separate commit.

File: .rubocop.yml (hunk 1 above)
Comment (chills42) [reply .rubocop.yml]: That was accidental, I was actually just fixing that.
